### ISSUE TITLE
Polygons bleeding into adjacent tiles

### ISSUE THREAD

**Issue by DougLau** [DougLau/earthwyrm#2]
At zoom levels > 16, polygons can bleed into adjacent tiles as corrupt shapes where they shouldn't be.  This can be seen in the segment layers for honeybee.

**Comment by eugenever** [comment]
First of all, thank you for a good library.
I worked with it a little and found the following pattern: if you increase `edge_extent` when increasing the `zoom,` then this bug goes away.

For debugging purposes, the [tile_config](https://github.com/DougLau/earthwyrm/blob/6e3d8b4b98e536ca83b6c92a6e8f3a55fda529d7/earthwyrm/src/tile.rs#L199) method was rewritten as follows:
```
/// Create tile config for a tile ID
fn tile_config(&self, tid: TileId) -> TileCfg {
    let tile_extent = self.tile_extent;
    let mut bbox = self.grid.tile_bbox(tid);
    // increase bounding box by edge extent
    let zoom = tid.z();
    let edex = if zoom < 13 {
        f64::from(self.edge_extent)
    } else if zoom < 15 && zoom >= 13 {
        f64::from(tile_extent / 3)
    } else if zoom <= 17 && zoom >= 15 {
        f64::from(tile_extent / 2)
    } else {
        tile_extent as f64
    };
    let edge = edex / f64::from(tile_extent);
    let edge_x = edge * (bbox.x_max() - bbox.x_min());
    let edge_y = edge * (bbox.y_max() - bbox.y_min());
    bbox.extend([
        (bbox.x_min() - edge_x, bbox.y_min() - edge_y),
        (bbox.x_max() + edge_x, bbox.y_max() + edge_y),
    ]);
    let ts = f64::from(tile_extent);
    let transform = self.grid.tile_transform(tid).scale(ts, ts);
    TileCfg {
        tile_extent,
        tid,
        bbox,
        transform,
    }
}
```

**Comment by DougLau** [comment]
Wow, thanks!  Maybe it makes sense to just get rid of the edge_extent parameter and make it always automatic.

**Comment by eugenever** [comment]
I agree with you, but it seems to me that it is worth limiting its maximum value according to some criterion (for example, no more than `tile_extent`). At first glance, values ​​of the `edge_extent` parameter should not be allowed to be too large.

**Comment by DougLau** [comment]
How about something like this?

```rust
/// Calculate edge ratio based on tile zoom
fn zoom_edge(tid: TileId) -> f64 {
    match tid.z() {
        0..=12 => 1.0 / 32.0,
        13 => 1.0 / 16.0,
        14 => 1.0 / 8.0,
        15 => 1.0 / 4.0,
        16 => 1.0 / 2.0,
        _ => 1.0,
    }
}
```

**Comment by eugenever** [comment]
Yes fine! I tested 21 zooms on my data, there were no corrupt shapes. At the same time, the client (browser) consumes little memory and CPU resources

**Comment by DougLau** [comment]
Awesome!  Thanks for the feedback

**Comment by DougLau** [comment]
Fixed in 615850e1d5affa72b4e41f91b9a98109bef76f6f
